src/main/java/com/google/cloud/spanner/pgadapter/parsers/NumericParser.java

```diff
@@ -23,6 +23,7 @@
 import org.postgresql.util.ByteConverter;
 
 /** Translate from wire protocol to {@link Number}. */
+// TODO: Change this to use String as type, and only convert to BigDecimal for binary transfer.
```

Comment:
Should we attach buganizer ids to these for making them easier to track and prioritise?

Reply:
Good point. I've added b/230559811 and referenced in the TODO in the code.